src/bittybuzz/include/bittybuzz/BittyBuzzUserFunctions.h

```diff
@@ -21,39 +21,49 @@ namespace BittyBuzzUserFunctions {
 
     /**
      *@brief register a new function, exposing it to the remote composant of the swarm
-     *@details This closure expects four parameters, one stringId (name of the function), one
-     * closure (the function itself), one table with the description of the arguments, and one with
-     *the self context
-     * The table description of the arguments is an array of tuple, the tuple has the arg name as
-     *key and the type, you can use any value, only the type is checked. You could define a variable
-     *as int=0 float=0.0 And use those variables for types description
+     *@details This closure expects four parameters, the closure can be a lambda.
+     * -# stringId (name of the function)
+     * -# a closure (the callback function)
+     * -# a table with the description of the arguments
+     * -# one with the self context
+     *
+     * The table description of the arguments is an array of tuple, the
```

Comment:
tuple"s"

Reply:
"... is an array of tuples, which hold the arg name as key and a value that can be of any type. Only the type is checked."
would probably be better